HtmlCssSample.css

```diff
@@ -0,0 +1,199 @@
+.main-body {
+    background-color: rgb(35, 56, 76);
+    display: flex;
+    margin: 0;
+}
+
+.navigation-bar {
+    position: fixed;
+    justify-content: space-between;
+    background-color: white;
+    height: 7%;
+    width: 100%;
+    display: flex;
+}
+
+.logo {
+    display: flex;
+    justify-content: flex-start;
+    margin-left: 2%;
+    margin-top: 1%;
+    width: 10%;
+    height: 50%;
+}
+
+.webpage-main-logo {
+    height: 100%;
+}
+
+.navigation-element {
+    display: inline-flex;
+    justify-content: space-evenly;
+    width: 60%;
+    height: 100%;
+}
+
+.navigation-links {
+    display: flex;
+    justify-content: flex-end;
+    list-style-type: none;
+    margin-top: 2%;
+    width: 65%;
+    height: 30%;
+}
+.navigation-links-element
+{
+    display: inline-block;
+    padding-right: 2%;
+    padding-left: 2%;
+}
+.navigation-links-element:hover {
+    background-color: rgb(235, 233, 233);
+    cursor: pointer;
+}
+
+.navigation-bar-buttons {
+    margin-top: 2%;
+    width:20%;
+    display: inline;
+}
+
+.btn-sign-in {
+    border-radius: 5px;
+}
+
+.btn-register {
+    border-radius: 5px;
+    background-color: black;
+    color: white;
+}
+
+.btn-sign-in:hover {
+    cursor: pointer;
+}
+
+.btn-register:hover {
+    cursor: pointer;
+}
+.main-contents {
+    display: flex;
+    flex-direction: column;
+    margin-top: 10%;
+    align-items: center;
+    width: 100%;
+    height: 100%;
+}
+.headings {
+    padding-left: 1%;
+    padding-bottom: 2%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.html-heading {
+    font-size: 4em;
+    color: white;
+}
+.css-heading {
+    font-size: 3em;
+    color: white;
+}
+.form-body {
+    background-color: rgb(245, 245, 245);
+    width: 20%;
+    height: 80%;
+    border-radius: 10px;
+    box-shadow: 10px 5px 5px rgb(131, 130, 130);
+    padding-top: 1%;
+}
+.form-outline {
+    border: 0;
+}
+.input-details {
+    display: flex;
+    width: 90%;
+    margin-left: 3%;
+    margin-bottom: 10%;
+    border-radius: 5px;
+    padding-top: 3%;
```

Comment:
instead of separate top and bottom use the padding instead

Reply:
resolved